src/eredis_cluster_pool.erl

```diff
@@ -36,6 +37,25 @@ create(Host, Port, Options) ->
             {ok, PoolName}
     end.
 
+-spec get_existing_pool(Host :: string() | binary(),
+                        Port :: inet:port_number()) ->
+          {ok, Pool :: atom()} | {error, no_pool}.
+get_existing_pool(Host, Port) when is_binary(Host) ->
+    get_existing_pool(binary_to_list(Host), Port);
+get_existing_pool(Host, Port) when is_list(Host) ->
+    try list_to_existing_atom(Host ++ "#" ++ integer_to_list(Port)) of
```

Comment:
Maybe use/change eredis_cluster_pool:get_name() to have the the name-specification in one place

Reply:
Sure, will do. I guess it's safe to use `list_to_atom` in this case. (I wanted to avoid creating atoms if the pool nor the atom exists, but that's a corner case and the Redis cluster is trusted.)